libraries/AP_Baro/AP_Baro_MS5611.h

```diff
@@ -88,19 +88,19 @@ class AP_Baro_MS56XX : public AP_Baro_Backend
     AP_SerialBus *_serial;
 
     /* Asynchronous state: */
-    volatile bool            _updated;
+    volatile bool            _updated = false;
```

Comment:
Actually removing from the constructor's initializer list and putting it here wouldn't increase the size. However we can just remove them and reduce the size a little bit.